www/data/docs_sidebar.yml

```diff
@@ -44,9 +44,9 @@ sidebar_links:
       link: "/docs/developing-packages/#debug-builds"
     - title: Package Export Formats
       link: "/docs/developing-packages/#pkg-exports"
-    - title: Share via Builder
+    - title: Share with Builder
```

Comment:
**TRIGGER WARNING**

Totally pedantic English thing that you can feel 100% free to ignore but I figured I'd share - Share _with_ builder feels like a header about sharing things to builder as opposed to _through_ builder. But perhaps "Publishing with Builder" alleviates any syntactic questions around the header?

Reply:
Publishing and sharing are totally different verbs from builder's perspective. Publishing is taking a Habitat artifact and publishing it to an alternate location, e.g. DockerHub. At some point we will have separate logs carved out to show this phase.